### PR TITLE
scan old namespace for backward compatibility

### PR DESCRIPTION
**Pull request by mesut**
It is better to scan both namespaces which are "org.zalando" and "de.zalando" to not force library user to change their package structure.

It fixes https://github.com/zalando-stups/java-sproc-wrapper/issues/67

### CODE REVIEW

File: src/test/java/de/zalando/OrderMonetaryAmountTransformer.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-package org.zalando.sprocwrapper.example.transformer;
+package de.zalando;
```
Comment (vadeg) [inline src/test/java/de/zalando/OrderMonetaryAmountTransformer.java:2]: Why did you change the package here?

**Review by vadeg — COMMENTED** [review]
Thanks for the PR. The suggested solution is more like a workaround. There is a `global.value.transformer.search.namespace` environment variable which specifies where which namespace to scan. If it is not set then it fallbacks to `org.zalando`. Now the variable accepts a string, but can be converted to accept an array of strings. I think it would be much cleaner. What do you think?

File: src/test/java/de/zalando/OrderMonetaryAmountTransformer.java (hunk 1 above)
Comment (mesut) [reply src/test/java/de/zalando/OrderMonetaryAmountTransformer.java:2]: Thank you for reviewing. I think we need to implement this workaround in order to not break clients. We can convert namespace variable to array. In this case we will expect from the clients to pass old namespace in **global.value.transformer.search.namespace**. And as a client of this library I was not aware of global.value.transformer.search.namespace until analyze the source code. 
For long story short, let's  make **global.value.transformer.search.namespace** an array and put some information in README.md ? What do think ? If you are ok with that, I will change the PR

Comment (vadeg) [reply src/test/java/de/zalando/OrderMonetaryAmountTransformer.java:2]: Yes, sounds good!
